src/NetAnalyzers/RulesMissingDocumentation.md

```diff
@@ -2,8 +2,6 @@
 
 Rule ID | Missing Help Link | Title |
 --------|-------------------|-------|
-CA1311 | <https://learn.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1311> | Specify a culture or use an invariant version |
-CA1421 | <https://learn.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1421> | This method uses runtime marshalling even when the 'DisableRuntimeMarshallingAttribute' is applied |
```

Comment:
I assume these changes are appropriate and happened automatically from running the tool?

Reply:
Right, it is happened automatically from running the tool, docs for these analyzers merged a few weeks ago.